hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -920,6 +857,41 @@ function upgrade_1018() {
     return true;
   }
 
+  /**
+   * Create civicrm_hrpay_scale table and its default data if it is not exist
+   *
+   */
+  function upgrade_1019() {
+    CRM_Core_DAO::executeQuery("
+        CREATE TABLE IF NOT EXISTS `civicrm_hrpay_scale` (
+        `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+          `pay_scale` VARCHAR(63) DEFAULT NULL,
+          `pay_grade` VARCHAR(63) DEFAULT NULL,
+          `currency` VARCHAR(63) DEFAULT NULL,
+          `amount` DECIMAL(10,2) DEFAULT NULL,
+          `periodicity` VARCHAR(63) DEFAULT NULL,
+          `is_active` tinyint(4) DEFAULT '1',
+          PRIMARY KEY(id)
+        ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=1
+      ");
+
+    $data = CRM_Core_DAO::singleValueQuery("SELECT COUNT(*) FROM `civicrm_hrpay_scale`");
+
+    if (empty($data)) {
+      CRM_Core_DAO::executeQuery("
+        INSERT INTO `civicrm_hrpay_scale` (`pay_scale`, `pay_grade`, `currency`, `amount`, `periodicity`, `is_active`) VALUES
+        ('US', 'Senior', 'USD', 38000, 'Year', 1),
+        ('US', 'Junior', 'USD', 24000, 'Year', 1),
+        ('UK', 'Senior', 'GBP', 35000, 'Year', 1),
+        ('UK', 'Junior', 'GBP', 22000, 'Year', 1),
+        ('Not Applicable', NULL, NULL, NULL, NULL, 1)
+    ");
+      return true;
+    }
+
+    return false;
```

Comment:
Is this return false ok? The upgrader method will return false if there pay scales, right? Meaning that the upgrade will stop in that case. Is that the expected behavior? What if we have other upgrader methods to be executed after this one?